rclcpp/include/rclcpp/publisher.hpp

```diff
@@ -116,6 +121,10 @@ class PublisherBase
   const rcl_publisher_t *
   get_publisher_handle() const;
 
+  RCLCPP_PUBLIC
+  const std::vector<std::shared_ptr<QOSEventHandlerBase>> &
```

Comment:
In headers, please use full type names, e.g. `rclcpp::QOSEventHandlerBase`.

Reply:
Done.